router/extension/error_handler.go

```diff
@@ -42,6 +43,10 @@ func ErrorHandler(logger *zap.Logger) echo.HTTPErrorHandler {
 			logger.Error(err.Error(), append(err.Fields, zap.String("requestId", GetRequestID(c)))...)
 			code = http.StatusInternalServerError
 			body = echo.Map{"message": http.StatusText(http.StatusInternalServerError)}
+		case *net.OpError:
+			logger.Warn("network error", zap.Error(err), zap.String("requestId", GetRequestID(c)))
```

Comment:
↓で充分かなって思ったんだけど、ここでリテラルにしてるのはどういう意図かちょっと気になります
多分`err.Error()`だと呼び出し元関数のエラー文がそのまま返ってくるよね…？
```suggestion
			logger.Warn(err.Error(), zap.Error(err), zap.String("requestId", GetRequestID(c)))
```

Reply:
下に書いてあったWarnログを真似しました。何か意図があるわけではないです。
https://github.com/traPtitech/traQ/blob/10cb4693d31b0d3285dd92755dc852ebc536df4d/router/extension/error_handler.go#L51-L60

Reply:
あ、`zap`のところに`err`がそのまま渡されてるし、内容が消し飛んでるわけではないのか
今気づきました
いいと思います！